frame/assets/runtime-api/src/lib.rs

```diff
@@ -4,17 +4,20 @@
 
 use codec::Codec;
 use composable_support::rpc_helpers::SafeRpcWrapper;
-
+use composable_traits::assets::Asset;
+use sp_std::vec::Vec;
 // Here we declare the runtime API. It is implemented it the `impl` block in
 // runtime amalgamator file (the `runtime/src/lib.rs`)
 sp_api::decl_runtime_apis! {
-	// REVIEW(benluelo): Should the AssetId type parameter be removed and then just use CurencyId directly?
+	// REVIEW(benluelo): Shou ld the AssetId type parameter be removed and then just use CurencyId directly?
```

Comment:
Looks like IDE has broke the word `Should`

Reply:
should be IDE